medusa/helpers/utils.py

```diff
@@ -61,3 +64,15 @@ def truth_to_bool(value):
     :return: boolean value, either True or False
     """
     return bool(strtobool(str(value))) if value else False
+
+
+def to_timestamp(dt):
+    """Return POSIX timestamp corresponding to the datetime instance.
+
+    :param dt: datetime (possibly aware)
+    :return: seconds since epoch as float
+    """
+    epoch = datetime(1970, 1, 1, tzinfo=tz.gettz('UTC'))
+    dt_utc = dt.astimezone(tz.gettz('UTC'))
```

Comment:
What would happen if it isn't aware?
If possible maybe wrap in `try/except` with the correct exception class, and put an error in the log.
I think timezone-aware `datetime`s is good to have.